### PR TITLE
Update the information in the About Box

### PR DESCRIPTION
**Pull request by phillip-hopper**
Update copyright date and links to other sites

---
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/saymore/89)

### CODE REVIEW

File: DistFiles/aboutBox.htm [hunk 1]
```diff
@@ -1,54 +1,60 @@
...
-    <p><a href="http://www.nunit.org">NUnit</a> Copyright © 2002-2012 Charlie Poole, Copyright © 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov, Copyright © 2000-2002 Philip A. Craig (<a href="http://nunit.org/index.php?p=license&r=2.6.2">Nunit License</a>)</p>
-    <p><a href="https://github.com/sillsdev/libpalaso">Palaso Library</a> © <a href="http://sil.org">SIL International</a> (<a href="http://sil.mit-license.org/">MIT</a>)</p>
-    <p><a href="https://bitbucket.org/hatton/l10nsharp">L10N Sharp</a> Localization library: © <a href="http://sil.org">SIL International</a> (<a href="http://sil.mit-license.org/">MIT</a>) </p>
+    <p><a href="https://github.com/naudio/NAudio">NAudio</a> Mark Heath (Copyright unknown) (<a href="https://github.com/naudio/NAudio/blob/master/license.txt">MS-PL</a>)</p>
+    <p><a href="https://github.com/nunit/nunit">NUnit</a> Copyright © 2002-2018 Charlie Poole, Copyright © 2002-2004 James W. Newkirk, Michael C. Two, Alexei A. Vorontsov, Copyright © 2000-2002 Philip A. Craig (<a href="https://github.com/nunit/nunit/blob/master/LICENSE.txt">Nunit License</a>)</p>
```
Comment (tombogle) [inline DistFiles/aboutBox.htm]: This is what I came up with for NUnit:
`<p><a href="https://nunit.org/">NUnit</a> Copyright © 2018 Charlie Poole, Rob Prouse (<a href="https://raw.githubusercontent.com/nunit/nunit/master/LICENSE.txt">Nunit License</a>)</p>`
based on the info from the NuGet package.

File: DistFiles/aboutBox.htm [hunk 2]
```diff
@@ -1,54 +1,60 @@
...
+    <p><a href="https://github.com/sillsdev/libpalaso">Palaso Library</a> © <a href="https://www.sil.org/">SIL International</a> (<a href="https://sil.mit-license.org/">MIT</a>)</p>
+    <p><a href="https://github.com/sillsdev/l10nsharp">L10N Sharp</a> Localization library: © <a href="https://www.sil.org/">SIL International</a> (<a href="https://sil.mit-license.org/">MIT</a>) </p>
```
Comment (tombogle) [inline DistFiles/aboutBox.htm]: I think we also use:

`<p><a href="https://www.newtonsoft.com/json">Newtonsoft.Json</a> Copyright © 2018 James Newton-King (<a href="https://www.nuget.org/packages/Newtonsoft.Json/12.0.1/license">MIT</a>)</p>`

File: DistFiles/aboutBox.htm [hunk 3]
```diff
@@ -1,54 +1,60 @@
...
     <p>Sarah Moeller (Graduate Institute of Applied Linguistics)</p>
-    <p>JetBrains for free open-source license for <a href="http://www.jetbrains.com/resharper/" data-mce-href="http://www.jetbrains.com/resharper/">Resharper</a>, which keeps our c# code clean and agile, <a href="http://www.jetbrains.com/teamcity/" data-mce-href="http://www.jetbrains.com/teamcity/">TeamCity</a>, which we use for continuous integration/builds, and <a href="http://www.jetbrains.com/webstorm/" data-mce-href="http://www.jetbrains.com/webstorm/">WebStorm</a>, for editing html and javascript.</p>
+    <p>JetBrains for free open-source license for <a href="https://www.jetbrains.com/resharper/" data-mce-href="https://www.jetbrains.com/resharper/">Resharper</a>, which keeps our C# code clean and agile, <a href="https://www.jetbrains.com/teamcity/" data-mce-href="https://www.jetbrains.com/teamcity/">TeamCity</a>, which we use for continuous integration/builds, and <a href="https://www.jetbrains.com/webstorm/" data-mce-href="https://www.jetbrains.com/webstorm/">WebStorm</a>, for editing html and javascript.</p>
```
[the remark this answers is not in the document]
Comment (phillip-hopper) [reply DistFiles/aboutBox.htm]: Done.

File: DistFiles/aboutBox.htm [hunk 4]
```diff
@@ -1,54 +1,60 @@
...
 </head>
 <body>
-    <h1>Copyright © 2011-2016 <a href="http://sil.org">SIL International</a></h1>
+    <h1 style="margin-top:8px">Copyright © 2011-2019 <a href="https://www.sil.org/">SIL International</a></h1>
 
     <h1>License</h1>
+    <p>Open source (<a href="https://sil.mit-license.org/">MIT</a>).</p>
 
-    <p>Open source (<a href="http://sil.mit-license.org/">MIT</a>).</p>
+    <h1>Website</h1>
```
[the remark this answers is not in the document]
Comment (phillip-hopper) [reply DistFiles/aboutBox.htm]: Done.

File: DistFiles/aboutBox.htm [hunk 5]
```diff
@@ -1,54 +1,60 @@
...
+    <p>JetBrains for free open-source license for <a href="https://www.jetbrains.com/resharper/" data-mce-href="https://www.jetbrains.com/resharper/">Resharper</a>, which keeps our C# code clean and agile, <a href="https://www.jetbrains.com/teamcity/" data-mce-href="https://www.jetbrains.com/teamcity/">TeamCity</a>, which we use for continuous integration/builds, and <a href="https://www.jetbrains.com/webstorm/" data-mce-href="https://www.jetbrains.com/webstorm/">WebStorm</a>, for editing html and javascript.</p>
 
-    <p>Microsoft, for inexpensive licenses of Visual Studio in the past and now free VS 2015 Community.</p>
+    <p>Microsoft, for inexpensive licenses of Visual Studio in the past and now free VS 2017 Community.</p>
```
[the remark this answers is not in the document]
Comment (phillip-hopper) [reply DistFiles/aboutBox.htm]: Done.

File: DistFiles/aboutBox.htm (hunk 1 above)
Comment (phillip-hopper) [reply DistFiles/aboutBox.htm]: Done.

File: DistFiles/aboutBox.htm (hunk 2 above)
Comment (phillip-hopper) [reply DistFiles/aboutBox.htm]: Done.

File: DistFiles/aboutBox.htm (hunk 4 above)
[the remark this answers is not in the document]
Comment (tombogle) [reply DistFiles/aboutBox.htm]: Add a 's' to the word "Website" and it's good to go.

**Pull request merged**